pokepro/src/components/Header/header.css

```diff
@@ -0,0 +1,24 @@
+header{
+    /* background-color: #6A5F94; */
+    margin-bottom: 20px;
+}
+
+.header__container{
+    display:flex;
+    justify-content: space-between;
+    height: 10vh;
+    width: 100vw;
```

Comment:
Better to use a fixed height in PX in this case since we are using flex on the page container.